docs/docsite/rst/playbook_guide/playbooks_reuse.rst

```diff
@@ -69,6 +69,8 @@ Task include and import statements can be used at arbitrary depth.
 
 You can still use the bare :ref:`roles <roles_keyword>` keyword at the play level to incorporate a role in a playbook statically. However, the bare :ref:`include <include_module>` keyword, once used for both task files and playbook-level includes, is now deprecated.
 
+Ansible runs all roles reused with ``include_role`` and ``import_role``, even if the parameters on the roles are the same. See :ref:`run_role_twice` for more details.
```

Comment:
The behavior on devel, which I believe is correct, runs 5 of the 8 role invocations, deduplicating the other 3, for this playbook:

```yaml
- hosts: localhost
  gather_facts: no
  roles:
    - name: role  # runs
    - name: role  # does not run (unless role has allow_duplicates: true in the meta/main.yml, the default is false)
    - name: role  # runs again regardless of the meta/main.yml value
      allow_duplicates: true
  tasks:
    # include_role and import_role do not use allow_duplicates in the meta/main.yml
    # the allow_duplicates parameter overrides it, and defaults to true for these
    
    - import_role:
        name: role  # runs (not duplicated with roles keyword even with allow_duplicates=false)
    - import_role:
        name: role  # runs again (allow_duplicates defaults to true)
    - import_role:
        name: role  # does not run
        allow_duplicates: false

    - include_role:
        name: role  # does not run (duplicated with import_role)
        allow_duplicates: false
    - include_role:
        name: role  # runs again
```